tap_klaviyo/client.py

```diff
@@ -90,17 +75,12 @@ def get_url_params(
             params.update(parse_qsl(next_page_token.query))
 
         if self.replication_key:
-            if self.get_starting_timestamp(context):
-                filter_timestamp = self.get_starting_timestamp(context)
-            elif self.config.get("start_date"):
-                filter_timestamp = _isodate_from_date_string(self.config("start_date"))
-            else:
-                filter_timestamp = DEFAULT_START_DATE
```

Comment:
drying code, the `get_starting_timestamp` will return the `stat_date` if no state is found.